titiler/api/endpoints/mosaic.py

```diff
@@ -0,0 +1,399 @@
+"""API for MosaicJSON Dataset."""
+import asyncio
+import os
+import random
+import re
+from functools import partial
+from typing import Dict, List, Optional
+from urllib.parse import urlencode
+
+import mercantile
+import morecantile
+import numpy
+import rasterio
+from cogeo_mosaic.backends import MosaicBackend
+from cogeo_mosaic.mosaic import MosaicJSON
+from cogeo_mosaic.utils import get_footprints
+from rio_tiler.io.cogeo import tile as cogeoTiler
+from rio_tiler.reader import point
+from rio_tiler.utils import geotiff_options
+
+from titiler.api import utils
+from titiler.api.deps import CommonMosaicParams, CommonTileParams
+from titiler.api.endpoints.cog import cog_info, tile_response_codes
+from titiler.errors import BadRequestError, TileNotFoundError
+from titiler.models.cog import cogBounds, cogInfo
+from titiler.models.mapbox import TileJSON
+from titiler.models.mosaic import CreateMosaicJSON, UpdateMosaicJSON
+from titiler.ressources.enums import (
+    ImageMimeTypes,
+    ImageType,
+    MimeTypes,
+    PixelSelectionMethod,
+)
+from titiler.ressources.responses import ImgResponse, XMLResponse
+
+from fastapi import APIRouter, Depends, Path, Query
+
+from starlette.concurrency import run_in_threadpool
+from starlette.requests import Request
+from starlette.responses import Response
+from starlette.templating import Jinja2Templates
+
+router = APIRouter()
+templates = Jinja2Templates(directory="titiler/templates")
+
+
+def _read_point(asset: str, *args, **kwargs) -> List:
+    """Read pixel value at a point from an asset"""
+    with rasterio.open(asset) as src_dst:
+        return point(src_dst, *args, **kwargs)
```

Comment:
Would you mind using rio-tiler-crs?

https://github.com/developmentseed/titiler/blob/725da5fa1bc56d8e192ae8ff0ad107493ca93378/titiler/api/endpoints/cog.py#L369-L370

Reply:
yeah i'll move everything over to rio-tiler-crs tomorrow